examples/dashboards/store.json

```diff
@@ -1339,7 +1339,7 @@
                "steppedLine": false,
                "targets": [
                   {
-                     "expr": "sum by (job, item_type) (rate(thanos_store_index_cache_requests_total{job=~\"$job\"}[$__rate_interval]))",
+                     "expr": "sum by (job, tenant, item_type) (rate(thanos_store_index_cache_requests_total{job=~\"$job\", tenant=~\"$tenant\"}[$__rate_interval]))",
```

Comment:
So I think changes from #6690 wouldn't break our mixin dashboards, as they are an extra label on series. It could break any custom dashboards that users have as noted in https://github.com/thanos-io/thanos/pull/6690#discussion_r1345742134 🙂 

Do you think these changes are needed here @jacobbaungard?

Reply:
From my understanding/testing, without these changes the dashboards should work as before the tenant changes, with graphs showing data from all tenants combined.

This change would break up graphs into a per tenant view, right? That could be what we want, but I mostly assumed most users won't be using the tenancy feature and therefore would prefer the graphs as they were before.

Reply:
I got rid of the mixins changes again; i can submit them just as regular PR later then!